week2-section2/exercise2/Song.cpp

```diff
@@ -0,0 +1,53 @@
+#include "Song.h"
+#include <iostream>
+#include <string>
+
+	FSong::FSong()
+	{
+		SongTitle = "";
+		ArtistName = "";
+		Duration = 0;
+	}
+
+	FSong::FSong (const std::string InSongTitle, const std::string InArtistName, const int InDuration)
+	{
+		SongTitle = InSongTitle;
+		ArtistName = InArtistName;
+		Duration = InDuration;
+	}
+
+	std::string FSong::GetSongTitle() const
+	{
+		return SongTitle;
+	};
+
+	std::string FSong::GetArtistName() const
+	{
+		return ArtistName;
+	};
+	
+	std::string FSong::GetFormatedDuration() const
+	{
+		int Minutes = Duration / 60;
+		int Seconds = Duration % 60;
+		if (Seconds > 10)
+		{
+			return std::to_string(Minutes) + ":" + std::to_string(Seconds);
+		}
+		else
+		{
+			return std::to_string(Minutes) + ":" +"0" + std::to_string(Seconds);
+		}
+		
+	};
+
+	bool FSong::bIsValid() const
```

Comment:
Functions don't need the b prefix. That's only for bool variables. The only rule for bool functions is for them to answer a true or false question, which IsValid does correctly :)